web-wallet/src/routes/(welcome)/setup/restore/NetworkSyncProgress.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+  import IconHeadingCard from "$lib/containers/Cards/IconHeadingCard.svelte";
+  import { ProgressBar } from "$lib/dusk/components";
+  import { walletStore } from "$lib/stores";
+  import { mdiCubeOutline } from "@mdi/js";
+  import { onDestroy } from "svelte";
+
+  $: ({ syncStatus } = $walletStore);
+
+  /** @type {boolean} */
+  export let isValid = false;
+
+  let syncStarted = false;
```

Comment:
Do we really need this extra flag?

Reply:
<img width="1894" alt="Screenshot 2024-08-26 at 14 13 21" src="https://github.com/user-attachments/assets/fa4e4fcb-c9e1-4d34-a0ff-5eae912d1d35">
<img width="1894" alt="Screenshot 2024-08-26 at 14 13 26" src="https://github.com/user-attachments/assets/2acdcaeb-805c-4c0c-908d-5045a0d36cd1">


Without it, it appears as if the sync has completed for a split second, before the sync actually starts.